openc3/lib/openc3/utilities/local_mode.rb

```diff
@@ -431,7 +431,7 @@ def self.sync_tool_config()
             # Parse just to ensure we have valid JSON
             JSON.parse(data, :allow_nan => true, :create_additions => true)
             # Only save if the parse was successful
-            ToolConfigModel.save_config(parts[-2], File.basename(config), data, scope: scope, local_mode: false)
+            ToolConfigModel.save_config(parts[-2], File.basename(config, '.json'), data, scope: scope, local_mode: false)
```

Comment:
Is this the fix for the weird config save where the name didn't have an extension?

Reply:
Config files were being synced in the init container with names like "myconfig.json".  
But when you save them in tools you just type myconfig.   It was causing a duplication with a different name at restart.  Config names now consistently don't include .json, but the files obviously do.